bcrypt-benchmark.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
```

Comment:
does not need to be in the top level directory to clutter other folders or source files